.github/ISSUE_TEMPLATE/broken-website.md

```diff
@@ -1,60 +1,57 @@
 ---
-name: Broken website
-about: Creates a broken website report about Dark Reader
-title: "[Broken Website] Replace with title"
+name: Broken Website Report
+about: Create a report about a website issue you experienced while using Dark Reader.
+title: "[Broken Website] Place title here"
 labels: Broken Website
 assignees: ''
 ---
 
-<!-- 
-  ⚠⚠ Do not delete this issue template! ⚠⚠ 
-  Issues that do not use the issue template/don't fill out the essential information are likely to be ignored and closed. 
+<!--
+⚠⚠ Do not delete this issue template! ⚠⚠
+Reported issues must use this template and have all the necessary information provided. Incomplete reports are likely to be ignored and closed.
 -->
 
 <!--
-  Thank you for taking the time to report a broken website.
-  Please make sure there is no existing issue with this broken website.
-  Make sure to fill out every section when needed else remove the section.
-  Don't forget to add the website in the issue title.
+Thank you for taking the time to create a report about a website issue.
+Ensure that there are no other existing reports for this website issue.
+Also, remember to fill out every section on this report and remove any
+that are not needed. Finally, place the website address in the title of
+this report.
 -->
 
-# Broken Website
-## Website
-<!-- The link to the website where you can observe the issue. -->
+# Broken Website Report
 
+## Website Address
+<!-- Provide the website address to allow us to troubleshoot this website issue. -->
 
-## How to Reproduce
-<!-- We need to know how you encountered the bug to properly troubleshoot the issue. -->
+## Steps To Reproduce
+<!-- Provide steps to reproduce to allow us to troubleshoot this website issue. -->
 <!--
-  An example of this is:
-  - Go to x site
-  - Hover over x button
-  - See that when hovering it isn't changing colors
+Here is an example:
+- Go to example.com.
+- Hover over the example button.
+- Notice that when hovering over the example button, it does not change color.
 -->
 
+## Expected Behavior
+<!-- Provide a clear and concise description of what you expected to happen. -->
 
-## Expected behavior
-<!-- A clear and concise description of what you expected to happen. -->
-
-
-## Actual behavior
-<!-- A clear and concise description of what actually happened. -->
-
+## Actual Behavior
+<!-- Provide a clear and concise description of what happened. -->
 
 ## Screenshots
-<!-- If applicable, add screenshots to help explain your problem. -->
-
+<!-- Add screenshots to help explain this website issue. -->
```

Comment:
I think we should preserve the `If applicable,` as not every issue would need it and I don't want screenshot when not needed.